### PR TITLE
Updated transform props

### PR DESCRIPTION
**Pull request by als-sdin**
Updated transform props for each component to match latest superset component calling convention.

Linter also made some small changes.

### CODE REVIEW

**Comment by als-sdin** [comment]
@grant-traynor Can you review this please.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 node_modules
 esm/
 lib/
+package-lock.json
```
Comment (grant-traynor) [inline .gitignore]: Not sure you want to do this?

File: packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js [hunk 2]
```diff
@@ -28,5 +25,5 @@ export default function transformProps(chartProps) {
     yAxisFormat,
     imageFile: imageSelectionControl,
     data: result,
-  }
+  };
```
Comment (grant-traynor) [inline packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js]: Having trouble following the rationale behind some of these changes. Is this the actual change?

File: .gitignore (hunk 1 above)
Comment (als-sdin) [reply .gitignore]: I'm relatively new to npm. I thought the package-lock was generated by npm when parsing package.json. I'll look into it

File: packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js (hunk 2 above)
Comment (als-sdin) [reply packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js]: No they have some kind of git linter that did it auto-magically. The only change I have made is renaming "payload" to "queryData".

Comment (als-sdin) [reply packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js]: I should have untangled the two changes.

File: .gitignore (hunk 1 above)
Comment (als-sdin) [reply .gitignore]: No you're right. I'll add it pack in.

Comment (grant-traynor) [reply .gitignore]: OK. Thanks for the explanation. Reckon we keep their linter in play if possible. Would be good to extend this sort of pre-commit QA across the organisation more broadly.

File: packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js (hunk 2 above)
Comment (grant-traynor) [reply packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js]: OK. No worries. That makes sense.

**Comment by grant-traynor** [comment]
Please go ahead and merge. Good work!

**Comment by grant-traynor** [comment]
Sorry. Think I closed the branch, was trying to close the pull request. Can you recover?

File: .gitignore (hunk 1 above)
Comment (als-sdin) [reply .gitignore]: Agreed. I will work with the linter and ensure our repo conforms with the next pull to master.

**Comment by als-sdin** [comment]
Done.

**Pull request merged**